bin/subgraphOperation.js

```diff
@@ -56,7 +56,7 @@ let networkMap = {
 };
 
 function selectConfig() {
-  let network = argv.network || "local";
+  let network = argv.network || "mainnet";
```

Comment:
Is this intentional?

Reply:
yes this is intentional the default is for a "mainnet" deployment locally.